### PR TITLE
feat(route): add Caa.reviews

### LINKED ISSUE #9823: 艺术网站rss
### 类型

科学期刊

### 网站地址

http://www.caareviews.org/

### 网站描述

CAA旗下的免费艺术学术文章网站

### 需要生成什么内容？

每个子栏目都可以提醒rss

或者

essays 和 book reviews 生成rss地址

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #9823

## 完整路由地址 / Example for the proposed route(s)

```routes
/caareviews/book
/caareviews/essay
/caareviews/exhibition
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [X] 新的路由 New Route
  - [X] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [X] 文档说明 Documentation
  - [X] 中文文档 CN
  - [X] 英文文档 EN
- [X] 全文获取 fulltext
  - [X] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/en/journal.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ Fill in parameter `query` with content after `http://export.arxiv.org/api/query?
 
 <RouteEn author="nczitzk" example="/bioone/journals/acta-chiropterologica" path="/bioone/journals/:journal?" :paramsDesc="['Journals, can be found in URL']"/>
 
+## Caa.reviews
```
Comment (TonyRL) [inline docs/en/journal.md:48]:
```suggestion
## caa.reviews
```

File: docs/en/journal.md [hunk 2]
```diff
@@ -45,6 +45,21 @@ Fill in parameter `query` with content after `http://export.arxiv.org/api/query?
...
+
+### Exhibition Reviews
+
+<RouteEn author="Fatpandac" example="/caareviews/exhibition" path="/caareviews/exhibition"/>
+
+### Essays
+
+<RouteEn author="Fatpandac" example="/caareviews/essay" path="/caareviews/essay"/>
+
+
```
Comment (TonyRL) [inline docs/en/journal.md:62]:
```suggestion

```
remove extra line break

File: docs/journal.md [hunk 3]
```diff
@@ -45,6 +45,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/bioone/journals/acta-chiropterologica" path="/bioone/journals/:journal?" :paramsDesc="['期刊名，可在期刊页地址栏中找到']"/>
 
+## Caa.reviews
```
Comment (TonyRL) [inline docs/journal.md:48]:
```suggestion
## caa.reviews
```

File: lib/v2/caareviews/book.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const { rootUrl, getList, getItems } = require('./utils');
+
+module.exports = async (ctx) => {
+    const url = `${rootUrl}/reviews/book`;
+
+    const list = await getList(url);
+    const items = await getItems(ctx, list);
+
+    ctx.state.data = {
+        title: `Book Reviews`,
```
Comment (TonyRL) [inline lib/v2/caareviews/book.js:10]:
```suggestion
        title: 'Book Reviews',
```

File: lib/v2/caareviews/essay.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const { rootUrl, getList, getItems } = require('./utils');
+
+module.exports = async (ctx) => {
+    const url = `${rootUrl}/reviews/essay`;
+
+    const list = await getList(url);
+    const items = await getItems(ctx, list);
+
+    ctx.state.data = {
+        title: `Essays`,
```
Comment (TonyRL) [inline lib/v2/caareviews/essay.js:10]:
```suggestion
        title: 'Essays',
```

File: lib/v2/caareviews/exhibition.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+const { rootUrl, getList, getItems } = require('./utils');
+
+module.exports = async (ctx) => {
+    const url = `${rootUrl}/reviews/exhibition`;
+
+    const list = await getList(url);
+    const items = await getItems(ctx, list);
+
+    ctx.state.data = {
+        title: `Exhibition Reviews`,
```
Comment (TonyRL) [inline lib/v2/caareviews/exhibition.js:10]:
```suggestion
        title: 'Exhibition Reviews',
```

File: lib/v2/caareviews/radar.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    'caareviews.org': {
+        _name: 'Caa Reviews',
```
Comment (TonyRL) [inline lib/v2/caareviews/radar.js:3]:
```suggestion
        _name: 'caa.reviews',
```

File: lib/v2/caareviews/utils.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+                const coverUrl = new URL($('div.cover > a').attr('href'), rootUrl).href;
+                const content = $('div.content.full-review').html();
+                item.description = art(path.join(__dirname, 'templates/utils.art'), {
+                    coverUrl,
+                    content,
+                });
+                $('div.review_heading').remove();
+                item.pubDate = parseDate($('div.header-text > div.clearfix').text());
+
```
Comment (TonyRL) [inline lib/v2/caareviews/utils.js:38]: Please also set `item.doi`

Comment (Fatpandac) [reply lib/v2/caareviews/utils.js:38]: This line aims to remove other content to get only time.

Comment (TonyRL) [reply lib/v2/caareviews/utils.js:38]: I mean this
https://github.com/DIYgod/RSSHub/blob/8a567d5cd4a7a10bb602dff4af55fb066e96d06a/lib/v2/elsevier/latest.js#L74-L75

Comment (Fatpandac) [reply lib/v2/caareviews/utils.js:38]: Get it. 😂

**Pull request merged**